### ISSUE TITLE
Bug - [Alert] - [aria-live="polite" not working on Windows]

### ISSUE THREAD

**Issue by ssilvert** [patternfly/patternfly#5219]
**Describe the problem**
See https://github.com/keycloak/keycloak-ui/issues/3176

Alerts do not get read to a user running NVDA screen reader on Windows.

**How do you reproduce the problem?**
You can reproduce using Keycloak Admin Console and running NVDA on Windows.  Any Alert that pops up won't be read by the screen reader.

**Expected behavior**
The Alert should be read to the user.  This does work on other operating systems.

**Is this issue blocking you?**
No, but it is an accessibility bug.

**What is your environment?**
 - OS: Windows
 - Browser: I've tried Chrome, FireFox, and Edge.  All exhibit the problem.

**Comment by mcoker** [comment]
cc @jessiehuff @thatblindgeye

**Comment by thatblindgeye** [comment]
Just tested with NVDA + Firefox and Chrome on Windows 10. If an alert will dynamically be added, having it render inside of an Alert Group, with `isLiveRegion` passed into the Alert Group, will get the alert to be announced. Currently that's the more recommended approach in our [Alert accessibility documentation](https://www.patternfly.org/v4/components/alert/accessibility), especially for toast alert groups. 

Though having the Alert itself be the live region _could_ work if the alert is already rendered on page load and its contents were to dynamically update.

**Comment by nicolethoen** [comment]
@ssilvert I agree with @thatblindgeye here, that patternfly officially recommends that dynamically added alerts should be placed in a live AlertGroup component, which exists on the page at page load. 

If you were to update the use case to use an AlertGroup and a live region poised to display a dynamic alert, does it then work in NVDA for you?

**Comment by thatblindgeye** [comment]
Will be closing this out since there hasn't been any activity for quite a while, and the issue should be resolved by comments above. If this is still an issue, however, please feel free to comment here or open another issue regarding this problem.
